files/image_config/monit/memory_checker

```diff
@@ -27,6 +27,11 @@ import re
 
 import docker
 
+from swsscommon import swsscommon
+
+EVENTS_PUBLISHER_SOURCE = "sonic-events-host"
+EVENTS_PUBLISHER_TAG = "mem-threshold-exceeded"
+events_handle = None
```

Comment:
>events_handle [](http://example.com/codeflow?start=0&length=13)

Better move to main function as local var.

Reply:
Done